### ISSUE TITLE
Packaging Application for Cross Platform Deployment

### ISSUE THREAD

**Issue by NamkhangNLe** [josephmasson26/DiagnoseMe#90]
Electron packaging is done, in testing phase to ensure that backend and frontend scripts start on open. Also not sure if the app even works when packaged as I get a white screen

**Comment by NamkhangNLe** [comment]
Will look at the link @josephmasson26 sent, I had originally used wine

**Comment by josephmasson26** [comment]
please do it in a branch!

**Comment by NamkhangNLe** [comment]
Yes Joseph. I did and merged it without testing and that's where the dilemma came from

**Comment by josephmasson26** [comment]
depending on how easy it is to do (and how many credits we buy), we could package a copy with .env and without for now until we build our middleware strategy
